### PR TITLE
Add message for moved issue

### PR DESCRIPTION
**Pull request by Marcono1234**
As discussed internally we should prefer moving issues to the correct project over resolving them as invalid, since that could lead to loss of information.

However, the different projects often have different fields and require different information. Therefore it is necessary to choose default values, which might be incorrect, during moving. This pull request adds a message explaining this.

As suggested internally, the issue should not necessary be resolved as "Awaiting Response" after having been moved, because that might cause it to fall through the filters of unprocessed issues. The message therefore does not mention that the report is or will be resolved.

### CODE REVIEW

**Comment by violine1101** [comment]
I feel like it sounds a bit... artificial? Technical? Don't know how to phrase it properly. But people might not know what to understand by "That project might require different or additional information".

I'd propose changing it to "As a result of the move, some information might have become lost or incorrect. *Please check whether the report is correct and complete, and if necessary, edit it accordingly.*" or something like that.

**Comment by Marcono1234** [comment]
Sounds good, though maybe an additional "still" could be added:
> As a result of the move, some information might have become lost or incorrect. \*Please check whether the report is **still** correct and complete, and if necessary, edit it accordingly.\*

**Review by osfanbuff63 — COMMENTED** [review]
I know, it's been over a year since this PR was updated, but I think this should probably replace the current `wrong-project` message (which still exists for some reason) instead of being its own message. I'll probably update this to match the current game status too.

I agree on rewording the line about the project might need different info as well.

File: assets/messages.yml [hunk 1]
```diff
@@ -1359,5 +1359,29 @@ messages:
...
+        - mce
+        - mcl
+        - mcpe
+        - realms
+      name: Wrong project (Moved)
+      shortcut: mov
+      message: |-
+        *Thank you for your report!*
+        
+        However, you have created the report for the wrong project, therefore it is being moved to the correct project. That project might require different or additional information, please check whether the report is correct and complete, *revise incorrect information and add missing information*.
```
Comment (Marcono1234) [inline assets/messages.yml:1378]: Should we use violine's wording then? For me that would be fine.
```suggestion
        However, you have created the report for the wrong project, therefore it is being moved to the correct project. As a result of the move, some information might have become lost or incorrect. *Please check whether the report is still correct and complete, and if necessary, edit it accordingly.*
```

File: assets/messages.yml [hunk 2]
```diff
@@ -1359,5 +1359,29 @@ messages:
...
         Please don't forget to search for an existing issue matching yours in the appropriate project before raising a new one.
 
+        %quick_links%
+      fillname: []
+  wrong-project-moved:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
```
Comment (Marcono1234) [inline assets/messages.yml:1369]: Minecraft Earth has been removed
```suggestion
```

**Review by Marcono1234 — COMMENTED** [review]
Thanks for updating this pull request!

> I think this should probably replace the current `wrong-project` message

Searching for that message on Mojira shows that it is still used in certain situations. So maybe we should keep both messages for now, or at least first discuss internally whether the other message can be removed.

> I'll probably update this to match the current game status too.

What do you mean by that?

File: assets/messages.yml (hunk 2 above)
Comment (osfanbuff63) [reply assets/messages.yml:1369]: This is what I meant mostly, removing unnecessary projects (I said games by mistake lol)

**Comment by osfanbuff63** [comment]
> > I'll probably update this to match the current game status too.
> 
> What do you mean by that?

Meant "projects" :P

File: assets/messages.yml (hunk 1 above)
Comment (osfanbuff63) [reply assets/messages.yml]: I think probably.

**Comment by Marcono1234** [comment]
Do you think there is anything else which should be improved? If not, I guess we could merge it and then inform the others internally about the newly added message (otherwise they might miss it).

**Comment by osfanbuff63** [comment]
> Do you think there is anything else which should be improved? If not, I guess we could merge it and then inform the others internally about the newly added message (otherwise they might miss it).

I have a draft message internally that I'd like to finish before merging this, but other than that this looks good I think.

**Review by SPGoding — APPROVED** [review]

**Review by osfanbuff63 — APPROVED** [review]
Never mind my message supplements this PR but doesn't need to be finished before this is merged.

**Comment by osfanbuff63** [comment]
@Marcono1234 we good on merging?

**Comment by Marcono1234** [comment]
Yes sorry, thanks for the review from both of you.

**Pull request merged**